includes/woopay/class-woopay-session.php

```diff
@@ -583,6 +584,22 @@ private static function create_woopay_nonce( int $uid ) {
 		return substr( wp_hash( $i . '|' . $action . '|' . $uid . '|' . $token, 'nonce' ), -12, 10 );
 	}
 
+	/**
+	 * Checks whether woopay is enabled. This is a duplicate of
+	 * WC_Payments_Features::is_woopay_eligible() except it doesn't rely on WC_Payments being initialized.
+	 *
+	 * @return bool
+	 */
+	private static function is_woopay_eligible() {
+		// Checks for the dependency on Store API AbstractCartRoute.
+		if ( ! class_exists( 'Automattic\WooCommerce\StoreApi\Routes\V1\AbstractCartRoute' ) ) {
+			return false;
+		}
+
+		// read directly from options to avoid loading WC_Payments.
+		return get_option( 'wcpay_account_data', [] )['data']['platform_checkout_eligible'] ?? false;
+	}
+
```

Comment:
On the same note as the previous comment: I think we can remove this function in favor of `WC_Payments_Feautres::is_woopay_eligibile()`.